lua/neo-tree/sources/filesystem/lib/fs_scan.lua

```diff
@@ -207,6 +207,10 @@ end
 
 local function get_children_async(path, callback)
   uv.fs_opendir(path, function(_, dir)
+    if err then
```

Comment:
Where does this `err` variable come from?

Reply:
oh oops. i messed up a different branch and copied the main part of the fix over but not all the changes. it should come from the first arg of the opendir callback.